### PR TITLE
chore: fix displayName / description

### PR DESCRIPTION
**Pull request by benoitf**
### What does this PR do?
use Podman AI Lab wording

### CODE REVIEW

**Comment by benoitf** [comment]
not sure for the second replacement

File: packages/backend/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "ai-lab",
-  "displayName": "AI Lab",
-  "description": "Podman Desktop AI Lab",
+  "displayName": "Podman AI Lab",
+  "description": "Podman AI Lab",
```
Comment (slemeur) [inline packages/backend/package.json:4]:
```suggestion
  "description": "Podman AI Lab lets you work with LLMs locally, exploring AI fundamentals, experimenting with models and prompts, and serving models while maintaining data security and privacy.",
```

**Review by slemeur — APPROVED** [review]

**Review by vrothberg — APPROVED** [review]
LGTM

**Review by jeffmaury — APPROVED** [review]
LGTM

**Pull request merged**